esmvalcore/cmor/tables/cmip6/Tables/CMIP6_Amon.json

```diff
@@ -1245,7 +1245,7 @@
             "units": "K", 
```

Comment:
Please do not change anything in the CMIP6 tables if at all possible. We may replace them with a newer version and all changes will be lost